src/Adapters/Storage/Mongo/MongoTransform.js

```diff
@@ -330,18 +330,11 @@ const parseObjectToMongoObjectForCreate = (className, restCreate, schema) => {
 // Main exposed method to help update old objects.
 const transformUpdate = (className, restUpdate, parseFormatSchema) => {
   let mongoUpdate = {};
-  let acl = addLegacyACL(restUpdate)._acl;
```

Comment:
Sorry I thought it was an intermittent test failure.
The tests were failing when restUpdate contained non-acl updates, e.g. `{ _failed_login_count: { __op: 'Increment', amount: 1 } }`.
I've restored the original code for line 334 which fixes the issue.